pkgs/top-level/perl-packages.nix

```
@@ -19213,6 +19213,18 @@ with self; {
       url = "mirror://cpan/authors/id/D/DT/DTOWN/Net-SNMP-v6.0.1.tar.gz";
       hash = "sha256-FMN7wcuz883H1sE+DyeoWfFM3P1epUoEZ6iLwlmwt0E=";
     };
+    patches = [
+      (fetchpatch {
+        name = "enable-newer-sha-algorithms.patch";
+        url = "https://raw.githubusercontent.com/openbsd/ports/refs/heads/master/net/p5-Net-SNMP/patches/patch-lib_Net_SNMP_Security_USM_pm";
```

The URL here should be pinned to a commit, maybe:
```suggestion
        url = "https://raw.githubusercontent.com/openbsd/ports/ab6365da3ac861e1c56520acdf388d421b357c17/net/p5-Net-SNMP/patches/patch-lib_Net_SNMP_Security_USM_pm";
```

---

Yes of course, my bad.

---

Fixed.